server/models.py

```diff
@@ -755,14 +765,11 @@ class Backup(Model):
     def can(cls, obj, user, action):
         if action == "create":
             return user.is_authenticated
-        if not obj:
+        elif not obj:
             return False
-        if user.is_admin:
+        elif user.is_admin or (action == "view" and user.id in obj.owners()):
```

Comment:
Do you think it makes sense to break the second part of this conditional into it's own `elif` branch? If we think of each conditional representing a specific case - the case of something being an admin is very different from being an owner of the object. 

I think quantifiedcode will be just as happy with 
```
elif user.is_admin
   return True 
elif action == "view"  and user.id in obj.owners():
   return True
```